File: 2-semestre/uri-online-judge/java/1759.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        int valor;
        
        Scanner entrada = new Scanner(System.in);
        
        valor = entrada.nextInt();
        
        for (int i = 0; i < valor; i++) {
            if (i == valor - 1) {
                System.out.printf("Ho!\n");
                break;
            }
            
            System.out.printf("Ho ");
        }
    }
}

File: 2-semestre/uri-online-judge/c/1759.c
#include <stdio.h>
 
int main() {
    int valor, i;
        
    scanf("%d", &valor);
    
    for (i = 0; i < valor; i++) {
        if (i == valor - 1) {
            printf("Ho!\n");
            break;
        }
        
        printf("Ho ");
    }
    
    return 0;
}
